<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emotion_DetectClass</class>
 <widget class="QMainWindow" name="Emotion_DetectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emotion_Detect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>325</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QSlider" name="video_status">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>398</width>
          <height>298</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="picshow">
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="choose_path">
       <property name="text">
        <string>Choose Path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QListWidget" name="fileList"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="predict_control">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Predict</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="but_loadlbf">
      <property name="text">
       <string>Load Model</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_start_detect">
      <property name="text">
       <string>Start Detect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_show_landmark">
      <property name="text">
       <string>Show LandMark</string>
      </property>
     </widget>
     <widget class="QPushButton" name="but_capture">
      <property name="text">
       <string>Stop/Start</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_res" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridWidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>19</y>
       <width>301</width>
       <height>100</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(216, 216, 216);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="4,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="label_res_1">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);
border-right-color: rgb(85, 85, 85);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_res_3">
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_res_2">
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);
border-right-color: rgb(85, 85, 85);
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_res_4">
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);
border-right-color: rgb(85, 85, 85);
border-bottom-color: rgb(85, 85, 85);
font: 75 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Resources/1_02.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_res_5">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);
border-right-color: rgb(85, 85, 85);
border-bottom-color: rgb(85, 85, 85);
font: 75 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Resources/1_01.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_res_0">
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);
border-right-color: rgb(85, 85, 85);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_res_6">
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);
border-right-color: rgb(85, 85, 85);
border-bottom-color: rgb(85, 85, 85);
font: 75 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Resources/1_03.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_res_7">
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
   border-style: solid;
   border-color: rgb(85, 85, 85,0);

border-bottom-color: rgb(85, 85, 85);
font: 75 14pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Resources/1_04.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_res_1</zorder>
     <zorder>label_res_2</zorder>
     <zorder>label_res_3</zorder>
     <zorder>label_res_0</zorder>
     <zorder>label_res_4</zorder>
     <zorder>label_res_5</zorder>
     <zorder>label_res_6</zorder>
     <zorder>label_res_7</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="model_info">
    <property name="geometry">
     <rect>
      <x>303</x>
      <y>230</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Model Info</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emotion_detect.qrc"/>
 </resources>
 <connections/>
</ui>
